yarn.lock

```diff
@@ -0,0 +1,5806 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Thanks! This PR looks great! We'd rather not introduce a lock file, as we tend to agree with this approach: 
> Lockfiles for apps, but not for packages.
https://github.com/sindresorhus/ama/issues/479#issuecomment-310661514